### PR TITLE
Readme Updates For Depends

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**
- more details on getting started
- Added License to readme
- Added Screenshot of tool
- Added Logo(s)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,75 @@
...
+```console
 dotnet tool install --global dotnet-depends
 ```
 
-# limitations
-* Currently only works with the new csproj format. See [issue #3](https://github.com/mholo65/depends/issues/3)
+Once installed, developers can analyze any project from a terminal instance.
+
+```console
+dotnet-depends ./depends.sln
```
Comment (mholo65) [inline README.md:24]: Nit: no need for dash

File: README.md [hunk 2]
```diff
@@ -1,10 +1,75 @@
...
+We can then install **Depends** using the following command.
+
+```console
+dotnet tool install dotnet-depends
+```
+
+To run the tool within a current solution, we need to run the following command.
+
+```console
+ dotnet dotnet-depends ./our-solution.sln
```
Comment (mholo65) [inline README.md:46]: Nit: no need for dash

**Review by mholo65 — COMMENTED** [review]
Wow! Thanks for this PR! Very much appreciated! And the logo looks awesome! Mind updating the NuGet package to also use the logo? Two minor nits.

File: README.md (hunk 1 above)
Comment (khalidabuhakmeh) [reply README.md:24]: suggest the fix, SUGGEST THE FIX!
```suggestion
dotnet depends ./depends.sln
```

File: README.md (hunk 2 above)
Comment (khalidabuhakmeh) [reply README.md:46]:
```suggestion
 dotnet depends ./our-solution.sln
```

**Pull request merged**